app/posts/manage-teacher-training-applications/2020-05-18-access-control-history.md

```diff
@@ -0,0 +1,551 @@
+
+Teacher training providers have to partner with ‘accredited bodies’ if they’re not an ‘accredited body’ themselves. Accredited bodies maintain the quality of teacher training providers.
+
+Different organisations need access to different information depending on what they’ve agreed with their partners.
+
+How can we go about achieving this in the Manage service?
+
+* [The initial problem statement](#the-initial-problem-statement)
+* [Context and guidance](#context-and-guidance)
+    * [How UCAS works](#how-ucas-works)
+    * [Research we've done](#research-we%E2%80%99ve-done)
+    * [User needs](#user-needs)
+    * [Options analysis session](#options-analysis-session)
+    * [Designs, research and iterations](#designs%2C-research-and-iterations)
+    * [Policy](#policy)
+    * [Next steps](#next-steps)
+    * [Current solution](#current-solution)
+    * [Worked examples](#worked-examples)
+* [Appendix - UCAS Weblink Screenshots](#appendix---ucas-weblink-screenshots)
+
+
+## The initial problem statement
+
+### Who are our users?
+People who manage applications. A user could be assigned to one or more providers in order to manage applications.
+
+Examples we know of include:
+* Head of admissions
+* Admissions officer
+* Administrator
+* Course lead
+* School lead
+* Subject matter specialist
+* Head of department
+* Course Tutor 
+
+Organisations can deliver initial teacher training manage through various organisational structures
+
+The most common organisational structures for providers are shown below:
+{{ appFigure({
+  image: {
+    file: "organisation-structures.png"
+  }
+}) }}
+
+### Why do we need to look into this?
+
+#### Hypotheses
+
+* There are situations in which provider users need different levels of access
+* Providers need to be able view applications made to their courses, or courses that they accredit
+* There should be commonality between Apply and Publish access designs
+* Access to a candidate’s sensitive information should be restricted
+
+#### Scenario 1- Situations in which provider users need different levels of access
+
+For many organisations, multiple different user types are involved in the application decision making process. We’ve identified common patterns through research where admissions staff will sift applications (on a daily basis), whilst academic staff will be largely involved in the interview process (may only require occasional access).
+
+#### Scenario 2 - managing applications across multiple organisations
+
+Some users manage applications across multiple provider organisations. Our current business rules state that provider users should be able to view and make decisions on applications to courses that they deliver, or ratify. The provider UI and API enable provider users to change the course as part of an offer. We need to make sure that if a course change happens, the application is no longer visible to the provider (if it is not a course they ratify, or deliver)
+
+##### Accredited bodies 
```

Comment:
I think headings get too far deeply nested by this point; can we think of a way of flattening the structure a bit?